### PR TITLE
DocumentDB: Expose EnableAutomaticFailover and ConsistentPrefix consistency policy

### PR DESCRIPTION
**Pull request by dmakwana**
- Switching to latest documentdb management pypi package 0.1.3.
- Exposing DatabaseAccount property "enableAutomaticFailover" in the create/update operations
- Exposing new consistency policy 'ConsistentPrefix'
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [X] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [X] Each command and parameter has a meaningful description.
- [] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by troydai** [comment]
For your reference: this is the code coverage for document db module: https://codecov.io/gh/Azure/azure-cli/pull/3153/tree/src/command_modules/azure-cli-documentdb/azure/cli/command_modules/documentdb

File: src/command_modules/azure-cli-documentdb/tests/test_documentdb_commands.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+class DocumentDBTests(ScenarioTest):
+    @ResourceGroupPreparer()
+    def test_create_database_account(self, resource_group):
+        name = self.create_random_name(prefix='cli', length=24)
+        self.cmd(
+            'az documentdb create -n {} -g {} '
+            '--enable-automatic-failover {} --default-consistency-level {}'
+            .format(name, resource_group, 'true', 'ConsistentPrefix'))
+        self.cmd('az documentdb show -n {} -g {}'.format(name, resource_group), checks=[
```
Comment (tjprescott) [inline src/command_modules/azure-cli-documentdb/tests/test_documentdb_commands.py]: Please also test your update command, as this has the new parameters.

File: src/command_modules/azure-cli-documentdb/tests/test_documentdb_commands.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            JMESPathCheck('properties.enableAutomaticFailover', 'true'),
+            JMESPathCheck('properties.consistencyPolicy.defaultConsistencyLevel',
+                          'ConsistentPrefix'),
+        ])
+        self.cmd('az documentdb list -g {}'.format(resource_group))
+        self.cmd('az documentdb list-keys -n {} -g {}'.format(name, resource_group))
+        self.cmd('az documentdb regenerate-key -n {} -g {} --key-kind {}'
+                 .format(name, resource_group, 'primary'))
+        self.cmd('az documentdb delete -n {} -g {}'.format(name, resource_group))
```
Comment (troydai) [inline src/command_modules/azure-cli-documentdb/tests/test_documentdb_commands.py]: Add proper verification to above four commands. Testing exit code only is not enough.

**Comment by dmakwana** [comment]
@troydai @tjprescott only testing changes that are introduced in this PR. Will add more comprehensive tests moving forward.

**Comment by troydai** [comment]
Thank you.

**Pull request merged**